src/three-domain-secure/component.jsx

```diff
@@ -115,6 +117,10 @@ export function getThreeDomainSecureComponent() : ZoidComponent<TDSProps> {
                 userType: {
                     type:     'string',
                     required: false
+                },
+                nonce: {
+                    type:     'string',
+                    required: true
```

Comment:
I think `required: true` is the default so we can remove that line all together.

What do you think about having this default to `getCSPNonce` similar to the other props in checkout-components?

```
type:    'string',
default: getCSPNonce
```

Reply:
Could the component be used from other projects or only through checkout-components?
If there is any chance, yes the default would be useful.

Reply:
> Could the component be used from other projects or only through checkout-components?

Yes it could be. Let's add this `default` option